src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AgreementsApiServiceImpl.scala

```diff
@@ -344,7 +344,8 @@ final case class AgreementsApiServiceImpl(
     consumerDocuments = agreement.consumerDocuments.map(_.toApi),
     createdAt = agreement.createdAt,
     updatedAt = agreement.updatedAt,
-    suspendedAt = agreement.suspendedAt
+    suspendedAt = agreement.suspendedAt,
+    existsContactMail = consumerTenant.mails.filter(_.kind == TenantProcess.MailKind.CONTACT_EMAIL).nonEmpty
```

Comment:
`exists` is more efficient
```suggestion
    existsContactMail = consumerTenant.mails.exists(_.kind == TenantProcess.MailKind.CONTACT_EMAIL)
```